packages/beacon-state-transition/src/allForks/util/attestation.ts

```diff
@@ -15,8 +15,8 @@ export function computeSubnetForAttestation(epochCtx: EpochContext, attestation:
  * Compute the correct subnet for a slot/committee index
  */
 export function computeSubnetForSlot(epochCtx: EpochContext, slot: number, committeeIndex: number): number {
-  const slotsSinceEpochStart = computeSlotsSinceEpochStart(slot);
-  const committeeCount = epochCtx.getCommitteeCountAtSlot(slot);
-  const committeesSinceEpochStart = committeeCount * slotsSinceEpochStart;
+  const slotsSinceEpochStart = slot % SLOTS_PER_EPOCH;
```

Comment:
Simpler and more efficient way to do the same thing. Also matches the spec function https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/validator.md#broadcast-attestation

```python
slots_since_epoch_start = uint64(slot % SLOTS_PER_EPOCH)
```